package.json

```diff
@@ -58,6 +58,7 @@
   },
   "devDependencies": {
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
+    "@types/cli-progress": "^3.11.5",
```

Comment:
To use `CommonUtils.loadJsonFromFile` I needed to include this or otherwise compiler complains @maliroteh-sf

Reply:
That's super weird. I don't know why the specific use of that method is triggering the issue.

The dependency tree for that package is entirely housed in an oclif _dev_ dependency in `lwc-dev-mobile-core`. It's a little disconcerting that we're dragging all of OClif along for the essentially generic Node.js utility functions that we leverage from `lwc-dev-mobile-core`. I'm taking a note to revisit this phenomenon in that package.

For the time being, the solution you chose is probably the easiest—that, or not using the method in favor of writing a local one in this project. At this time I don't know how we'd actually clear that error in the place it rightfully should be cleared.

Reply:
`CommonUtils` has 3 methods `startCliAction` / `updateCliAction` / `stopCliAction` that import `ux` from `@oclif/core`

I guess when trying to use `CommonUtils.loadJsonFromFile` the entire `CommonUtils` is pulled in and so you end up needing to add this dependency. In `lwc-dev-mobile-core` we have a dependency on `@oclif/core` which in turn pulls in `@types/cli-progress` so I guess what you're doing here is ok.

Reply:
I wonder why it's pulling in the entirety of `CommonUtils`, when other methods aren't. I wonder if it's because `loadJsonFromFile` returns `any`, maybe?